internal/testdata/dlaqr5test/main.go

```diff
@@ -0,0 +1,207 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Package level comment to explain what this does. Also comment on how you are handling row/col major between the two given that you're calling Fortran, but using the results in Go for the tests.